.editorconfig

```diff
@@ -0,0 +1,125 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
```

Comment:
Maybe a dumb question: is using tab a .NET idiom/rule/best practice/habit? I used to remember that Windows code style is much more about tab than spaces but wonder if it's still the case 😉

Reply:
It's a good question :). I don't have the response.

I can align to aspnetcore and use "space", if you want.

Reply:
If possible, yes please do so. Thanks!